src/AdoNet/Orleans.Persistence.AdoNet/Options/AdoNetGrainStorageOptions.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using Orleans.Persistence.AdoNet.Storage;
+using Orleans.Runtime;
+using Orleans.Runtime.Configuration;
+
+namespace Orleans.Hosting
+{
+    public class AdoNetGrainStorageOptions
+    {
+        /// <summary>
+        /// Connection string for AdoNet storage
+        /// </summary>
+        public string ConnectionString { get; set; }
+
+        /// <summary>
+        /// Stage of silo lifecycle where storage should be initialized.  Storage must be initialzed prior to use.
+        /// </summary>
+        public int InitStage { get; set; } = DEFAULT_INIT_STAGE;
+        /// <summary>
+        /// Default init stage in silo lifecycle
+        /// </summary>
+        public const int DEFAULT_INIT_STAGE = SiloLifecycleStage.ApplicationServices;
+
+        /// <summary>
+        /// The default ADO.NET invariant used for storage if none is given. 
+        /// </summary>
+        public const string DEFAULT_ADONET_INVARIANTS = AdoNetInvariants.InvariantNameSqlServer;
```

Comment:
It looks like this should be in singular, i.e. `public const string DEFAULT_ADONET_INVARIANTS`. It looks like this will in public, user facing API, so a .NET naming convention `DefaultAdoNetInvariant` would seem less extravagant.